tests/test_dimension.py

```diff
@@ -785,3 +785,32 @@ def test_symbolic_factor(self):
         assert np.all(usave.data[1] == 3)
         assert np.all(usave.data[2] == 5)
         assert np.all(usave.data[3] == 7)
+
+    def test_implicit_dims(self):
+        """
+        Test ConditionalDimension as an implicit dimension for an equation.
+        """
+
+        # This test makes an operator that should create a vector of increasing
+        # integers, but stop incrementing when a certain stop value is reached
```

Comment:
`.` at the end ?

Reply:
The other comments in other tests do not have a . at the end

Reply:
you can ignore this, @ArthurGiannota

we only enforce fullstops in numpydoc docstrings. Here it's irrelevant